workflow-examples/src/main/java/com/redhat/parodos/examples/prebuilt/migrationtoolkit/MigrationAssessmentWorkflow.java

```diff
@@ -86,10 +86,12 @@ WorkFlowOption defaultOption() {
 	@Bean(name = "AnalyzeApplicationAssessment")
 	@Assessment(parameters = {
 			@Parameter(key = "repositoryURL", description = "The repository with the code to analyze",
-					type = WorkParameterType.URL, optional = false),
+					type = WorkParameterType.TEXT, optional = false),
```

Comment:
why not URL?  because of the protocol that can be ssh? if that's the case I suggest to replace URL with URI which does support that, or at least behind the hood to use URI as the resolver

Reply:
Java URI support opaque schemes . `URI.create("foo://bar.baz").getScheme() // foo"`

Reply:
ok let's replace URL with URI in next pr.  it involves UI work too

Reply:
UI's pattern for url is "^(https?)://"